libflux/include/influxdata/flux.h

```diff
@@ -23,8 +23,6 @@ struct flux_error_t;
 void flux_free_error(struct flux_error_t *);
 
 // flux_error_str will return a string representation of the error.
```

Comment:
Can you add a message here that the string returned is a borrowed reference to the error and becomes invalid when the error is freed? I'd like that to be explicitly stated.